packages/tailwindcss-language-server/src/projects.ts

```diff
@@ -1029,6 +1052,18 @@ export async function createProjectService(
       state.classList = classList
       state.variants = getVariants(state)
 
+      let deps = designSystem.dependencies()
+
+      for (let dep of deps) {
+        dependencies.add(dep)
+      }
+
+      watchPatterns(Array.from(deps))
+
+      // Update capabilities to force color swatches to update across editors
+      // TODO: Need to verify how well this works across various editors
+      // updateCapabilities()
```

Comment:
Is this something you want to do before this PR is merged?

Reply:
No, I left this as a note for me to check soon but I don't want this to block the actual fix)